serveradmin/serverdb/querybuilder.py

```diff
@@ -58,13 +56,14 @@ def get_sql_condition(self, attribute, filt):
         template = ''
 
         if attribute.type == 'boolean':
-            if isinstance(filt, BaseFilter):
+            if type(filt) != BaseFilter:
```

Comment:
Don't really get that change? Why first check for BaseFilter and now contra BaseFilter?

Reply:
Ah, also because of the commit before? But then why checking it here explicitly and removing all other checks?

Reply:
Now every filter passed to this function are `isinstance(BaseFilter)`.  For booleans attributes the the must exactly be `BaseFilter` not a subclass of it.  `BaseFilter` works for exact matching.  Other complicated filters doesn't make sense on booleans.

Reply:
I am also adding a comment.